src/app/components/transcriptions/transcription-requests/transcription-requests.component.html

```diff
@@ -1 +1,22 @@
 <h1 class="govuk-heading-l">Transcript requests</h1>
+
+@if (transcriptRequests$ | async; as rows) {
+  <app-data-table id="transcript-requests-table" [rows]="rows" [columns]="columns">
```

Comment:
Would be good to get @deanbullock-hmcts 's [PR](https://github.com/hmcts/darts-portal/pull/571) in and add
```
noDataMessage="There are no outstanding transcript requests"
```

Reply:
Will be coming in the next commit

Reply:
Cool, was just going to comment that that PR is now merged 🎉